CMSgov/mint-app
# [NOREF] Revert changes that removed updates from #271

# NOREF

## Changes and Description

- https://github.com/CMSgov/mint-app/pull/284 overwrote code introduced from https://github.com/CMSgov/mint-app/pull/271
- This PR essentially puts #271 back into the codebase

## How to test this change

1. Navigate to the Model Plan and remove it
2. Ensure the modal has the changes ![image](https://user-images.githubusercontent.com/2349702/199997085-533c32e0-0f7a-413d-a0cd-10e04a78b9b8.png)
3. Ensure the `Success` prompt shows on the homepage ![image](https://user-images.githubusercontent.com/2349702/199997217-2192e7ee-54fe-441c-aa7b-70bdec3fa0f8.png)




## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
